src/class-elements/grammar-privatename-no-receiver-error-accessor-getter-meth.case

```diff
@@ -0,0 +1,23 @@
+// Copyright (C) 2018 Jaideep Bhoosreddy (Bloomberg LP). All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+desc: It is syntax error if IdentifierName is accessed without receiver (Accessor get Method)
+info: |
+  Updated Productions
```

Comment:
Just noticed this, "Updated Productions" is something that (sometimes) appears in proposal specs, but isn't part of the actual specification.